cdap-ui/app/cdap/components/Header/index.js

```diff
@@ -116,21 +116,37 @@ export default class Header extends Component {
     let dataprepBasePath = `${basePath}/dataprep`;
     let connectionsBasePath = `${basePath}/connections`;
     let rulesenginepath = `${basePath}/rulesengine`;
+    let analytics = `${basePath}/experiments`;
     if (
       location.pathname.startsWith(basePath) &&
       !location.pathname.startsWith(dataprepBasePath) &&
       !location.pathname.startsWith(connectionsBasePath) &&
-      !location.pathname.startsWith(rulesenginepath)
+      !location.pathname.startsWith(rulesenginepath) &&
+      !location.pathname.startsWith(analytics)
     ) {
       return true;
     }
     return false;
   };
 
+  isMMDSActive = (match, location = window.location) => {
+    if (match && match.isExact) {
+      return true;
+    }
+    let {selectedNamespace: namespace} = NamespaceStore.getState();
+    let experimentsBasePath = `/ns/${namespace}/experiments`;
+    if (location.pathname.startsWith(experimentsBasePath)) {
```

Comment:
nit: I guess we can just do `return location.pathname.startsWith(experimentsBasePath)`